models/l3afd.go

```diff
@@ -50,6 +50,8 @@ type BPFProgram struct {
 	ConfigArgs        L3afDNFArgs         `json:"config_args"`           // Map of arguments to config command
 	MonitorMaps       []L3afDNFMetricsMap `json:"monitor_maps"`          // Metrics BPF maps
 	EPRURL            string              `json:"ebpf_package_repo_url"` // Download url for Program
+	ObjectFile        string              `json:"object_file"`           // Object file contains kernel code
+	ProgramName       string              `json:"program_name"`          // BPF program name to load
```

Comment:
Line 27 has "Name of the BPF program".  What's the difference between that and "BPF program name to load"?